### ISSUE TITLE
341490 - [Newsroom][High][Content Migration]- Link redirected to PNF

### ISSUE THREAD

**Issue by veejayricana** [hlxsites/accenture-newsroom#407]
Steps to Reproduce: | Analysis & Proposed Solution: | Location of the Element:
Open URL
Legacy: https://newsroom.accenture.com/industries/automotive/accenture-to-demonstrate-fcas-uconnect-live-services-at-2015-mobile-world-congress.htm
Migrated: https://tempnewsroom.accenture.com/news/2015/accenture-to-demonstrate-fcas-uconnect-live-services-at-2015-mobile-world-congress
Click on link  Selected by FCA in EMEA to co-develop the technology and mobile platform
link : https://tempnewsroom.accenture.com/article_display.cfm?article_id=6118
Expected:
![image](https://github.com/hlxsites/accenture-newsroom/assets/143703040/d10bc7b0-21c0-4e3c-9b54-983c45a044f0)
Actual:
![image](https://github.com/hlxsites/accenture-newsroom/assets/143703040/afd614dc-e550-4620-9708-58cb98855c6e)
Azdo Link: https://dev.azure.com/accenturecio26/WebContentDigAssetMgmt_PR1093/_workitems/edit/341490
cc: @raymond-h-dev @ravkiran @amol-anand

**Comment by veejayricana** [comment]
Hi @amol-anand, and also do you have a way to check if it is the only page that has this kind of URL structure? 

cc: @raymond-h-dev

**Comment by amol-anand** [comment]
No have not seen those urls before. I ran a search on the content as well and only found this other one in EN. Also ran the search across GEOs and did not find anything there.

`find * -name "*.docx" -exec ~/Applications/docxtools {} _grep 'article_display' \;`

`import/news/2015/efma-and-accenture-open-application-period-for-third-annual-global-retail-banking-innovation-awards.docx: https://newsroom.accenture.com/article_display.cfm?article_id=6224
`
I've modified the urls to point to the right articles for both the pages and previewed/published them.
